docs/Contribute/ContributeDesign.md

```diff
@@ -0,0 +1,10 @@
+# Design BTCPay Server
+
+## Why designing matters
+
+Designing for BTCPay Server consists of creating and refining the user interface (UI) and user experience (UX) on a viewing standpoint, making it easier to navigate the menus, options, etc...
+
+To contribute design to BTCPay Server, check out the [UX/UI chat](https://chat.btcpayserver.org/btcpayserver/channels/ux-ui) for current discussions and the Figma [getting started](https://www.figma.com/file/C7Xyq0FlxgFW8vaBr8ht1z/BTCPAY?node-id=1766%3A1005) section for the design projects that are currently being worked on.
+
+Graphic content also helps promote BTCPay Server. Be it full scale logos, memes or other graphics, we're always enjoying seeng what contributors come up with.
+Feel free to check out the [Codified Design Guidlines](https://design.btcpayserver.org/) if you want consistency with graphic content created by other contributors.
```

Comment:
> if you want consistency with graphic content 

Not if you want, this is required, `for  graphic consistency with our design standards`.

Reply:
```suggestion
Following the [Codified Design Guidelines](https://design.btcpayserver.org/) is **required** so that graphical content can be consistent between contributors.
```